Source/engine.h

```diff
@@ -68,6 +68,16 @@ struct Point {
 	int x;
 	int y;
 
+	/**
+	 * @brief Converts a point to an SDL_Rect with 0 width/height

```

Comment:
This is... weird to me. A rect with 0 width and height shouldn't even be a valid rect.

Reply:
DrawString uses the SDL_Rect type as a dual use positioning and clipping rectangle, rectangles with size 0 effectively mean "don't clip, just position". This conversation was added as a hopefully temporary convenience instead of refactoring that function as part of this change and blowing out the diff.

Reply:
I'd rather we refactored that first and never had this in the first place.

Reply:
I'll add an overload of DrawString that takes a point instead of a rect when I get home, that will remove the only use of this conversion.